tests/Api/Shop/CustomersTest.php

```diff
@@ -138,6 +138,26 @@ public function it_sends_reset_password_email(): void
         $this->assertEmailCount(1);
     }
 
+    /** @test */
+    public function it_validates_wrong_reset_password_request(): void
+    {
+        $this->loadFixturesFromFiles(['authentication/customer.yaml', 'channel.yaml']);
+
+        $this->client->request(
+            method: 'POST',
+            uri: '/api/v2/shop/customers/reset-password',
+            server: self::CONTENT_TYPE_HEADER,
+            content: json_encode([
+                'email' => 'wrong_email',
+                'locale' => 'te_ST',
```

Comment:
```suggestion
                'localeCode' => 'te_ST',
```
as we can pass here either the locale code or the IRI of the locale